rascal-vscode-extension/src/test/vscode-suite/ide.test.ts

```diff
@@ -155,5 +156,21 @@ describe('IDE', function () {
         await driver.actions().doubleClick(mainItem!).perform();
         await driver.wait(async ()=> (await editor.getCoordinates())[0] === 5, Delays.normal, "Cursor should have moved to line that contains the println function");
     });
+
+    it ("rename works", async() => {
+        const editor = await ide.openModule(TestWorkspace.libFile);
+        await editor.moveCursor(7, 15);
+
+        await bench.executeCommand("Rename Symbol");
+        const renameBox = await ide.hasElement(editor, By.className("rename-input"), Delays.fast, "Rename box should appear");
+        await renameBox.sendKeys(Key.BACK_SPACE, "i", Key.ENTER);
```

Comment:
can we not use `setText` on the input box?

Reply:
As discussed in private, that's not straightforward here. Resolving.